editor.planx.uk/src/@planx/components/List/utils.tsx

```diff
@@ -21,6 +22,7 @@ export function formatSchemaDisplayValue(
 ) {
   switch (field.type) {
     case "number":
+      return field.data.units ? `${value} ${field.data.units}` : value;
```

Comment:
This is my only change ! 

Rest are formatting only, I guess prettier has changed with recent dependabot updates? Sorry for extra noise.

Reply:
Ah yep that makes sense - I'll open a `pnpn lint:fix` PR to fix this in a oner and try to remember this next time there's a prettier update 👍

Reply:
PR here - https://github.com/theopensystemslab/planx-new/pull/3372